### DISCUSSION TITLE
Create unix_timestamp_millis ?

### DISCUSSION THREAD

**Discussion by rejao** [time-rs/time#446] [Question]
I see we have the option for [unix_timestamp](https://github.com/time-rs/time/blob/ffd53f0c25599fd6c46060a524c900c09d5838b9/src/offset_date_time.rs#L213) and [unix_timestamp_nanos](https://github.com/time-rs/time/blob/ffd53f0c25599fd6c46060a524c900c09d5838b9/src/offset_date_time.rs#L232) for OffsetDateTime

Any reason why ```unix_timestamp_millis``` does not exist? It would be very useful

**Comment by jhpratt** [comment]
What's wrong with `foo.unix_timestamp_nanos() / 1_000_000`?

**Reply by rejao** [reply]
Just that the user would have to add this divide every time he needs the UTC in milliseconds. It would be better to have it in the library imo

**Reply by jhpratt** [reply]
Where is such a value needed, anyways?

**Reply by rejao** [reply]
I am migrating from chrono and use this quite often: https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.timestamp_millis

**Reply by jhpratt** [reply]
Use it where, though? That's what I'm asking.

**Comment by rejao** [comment]
The usage is mostly for elapsed time calculations (in milliseconds)

**Reply by jhpratt** [reply]
[`Duration::time_fn`](https://time-rs.github.io/api/time/struct.Duration.html#method.time_fn) seems like something you'd be interested in then. It's better than relying on the Unix timestamp.
